pypeit/tests/test_coadd.py

```diff
@@ -10,9 +10,10 @@
 from linetools.spectra.utils import collate
 from linetools.spectra.xspectrum1d import XSpectrum1D
 
-from pypeit.core import coadd
+from pypeit.core import coadd1d
```

Comment:
I know you're still developing but can you do the following:
 - For tests you consider obsolete because you're deprecating the old code, I would say that you can just remove them.
 - For tests that you think you'll keep but only test code that you're not finished with yet, add a TODO and comment out the test.
 - Fix the tests that test methods you think you're done with in the new code.

Reply:
I prefer to wait until the data model is finalized before writing the tests, as that will be more efficient. I'm happy to just move this test if you prefer that.

Reply:
I'm not sure how @profxj set up the settings for the develop branch.  It may have to pass Travis before it's allowed to be merged, which means you'll need to do something.  I'd prefer that you do what I suggested, but leave it to you to decide.

Reply:
comment out most tests and add a TODO.